Helper/PaymentDirectHelper.php

```diff
@@ -59,10 +59,10 @@ public function buildTransaction(UserInterface $userDebited, UserInterface $user
         return $transaction;
     }
 
-    public function executeDirectTransaction(UserInterface $userDebited, UserInterface $userCredited, $amount, $fees, $secureModeReturnURL = null)
+    public function executeDirectTransaction(UserInterface $userDebited, UserInterface $userCredited, $amount, $fees, $secureModeReturnURL = 'http://vago.local/app_dev.php/server-time')
```

Comment:
The vago.local url shouldn't exists in the mangopay bundle, could you remove it ?